src/main/java/com/cloudant/client/org/lightcouch/internal/CouchDbUtil.java

```diff
@@ -242,4 +246,17 @@ public static String ensureDesignPrefix(String id)
     {
         return id.startsWith(CouchDbUtil.DESIGN_PREFIX) ? id : CouchDbUtil.DESIGN_PREFIX + id;
     }
+
+    /**
+     * Remove `_design` prefix if it exists.
+     * The _design prefix is handled in {@link ViewQueryParameters#getViewURIBuilder()}
+     * @param id The design document's id
+     * @return String in the form of `id`
+     */
+    public static String removeDesignPrefix(String id)
```

Comment:
This `CouchDbUtil` class seems to be a bit of a mixed bag of utility methods which don't belong elsewhere. Is it more appropriate for `removeDesignPrefix` `ensureDesignPrefix` etc to live in `DatabaseURIHelper` which deals with manipulating URL paths?